COLLABORATOR_GUIDE.md

```diff
@@ -411,64 +411,45 @@ recommended but not required.
 
 ### Deprecations
 
-**Deprecation** refers to the identification of Public APIs that should no
-longer be used.
```

Comment:
I think it would be good to keep this. Especially non-native English speakers and people who are new to coding might not know what a deprecation stands for.

Reply:
I agree with @BridgeAR.

Reply:
PR to improve _Deprecation_ definition was posted at #20788